AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/MRECs/Programmatic/ALDemoProgrammaticMRECViewController.m

```diff
@@ -31,39 +31,70 @@ - (void)viewDidAppear:(BOOL)animated
-                                                              attribute: NSLayoutAttributeCenterX
-                                                              relatedBy: NSLayoutRelationEqual
-                                                                 toItem: self.view
-                                                              attribute: NSLayoutAttributeCenterX
-                                                             multiplier: 1.0
-                                                               constant: 0.0],
-                                 [NSLayoutConstraint constraintWithItem: self.adView
-                                                              attribute: NSLayoutAttributeCenterY
-                                                              relatedBy: NSLayoutRelationEqual
-                                                                 toItem: self.view
-                                                              attribute: NSLayoutAttributeCenterY
-                                                             multiplier: 1.0
-                                                               constant: 0.0],
-                                 [NSLayoutConstraint constraintWithItem: self.adView
-                                                              attribute: NSLayoutAttributeHeight
-                                                              relatedBy: NSLayoutRelationEqual
-                                                                 toItem: nil
-                                                              attribute: NSLayoutAttributeNotAnAttribute
-                                                             multiplier: 1.0
-                                                               constant: kMRECHeight],
-                                 [NSLayoutConstraint constraintWithItem: self.adView
-                                                              attribute: NSLayoutAttributeWidth
-                                                              relatedBy: NSLayoutRelationEqual
-                                                                 toItem: nil
-                                                              attribute: NSLayoutAttributeNotAnAttribute
-                                                             multiplier: 1.0
-                                                               constant: kMRECWidth]]];
+    
+    [self.view addConstraints: @[
+        [NSLayoutConstraint constraintWithItem: self.callbackTableView
+                                     attribute: NSLayoutAttributeTop
+                                     relatedBy: NSLayoutRelationEqual
+                                        toItem: self.view
+                                     attribute: NSLayoutAttributeTop
+                                    multiplier: 1.0
+                                      constant: 0],
+        [NSLayoutConstraint constraintWithItem: self.callbackTableView
```

Comment:
The constraints seem to be a little inconsistent between Swift/iOS (Swift does not have the -16.0 constant). I would avoid negative constants in general for constraint values as it's not as intuitive.

Reply:
made everything 0 for leading/trailing now on both programattic and storyboard

Reply:
One more negative constraint on L83. Same goes for the Swift counterpart.

Reply:
adjusted